web/themes/custom/votegov/templates/block/block-content--registration-form-selector.html.twig

```diff
@@ -35,7 +35,7 @@
   {% endif %}
 
   <form class="usa-form" method="get" id="register_form_download" data-test="nvrfForm">
-    <p>{{ content.field_description | field_value }}</p>
+    <p>{{ content.field_pdf_description | field_value }}</p>
```

Comment:
**Note:** If we made this change on prod it would remove the content that we currently have from the display. A better approach would be to add a fallback to the original field:
`{{ content.field_pdf_description | field_value | default(content.field_description | field_value) }}`

This way the new content field will override the existing one so there will always be content displaying.

Then when we remove the deprecated field we can also remove the fallback from this file.